Preliminary fix for dot corruption in pic_filename

i92guboj opened the pull request:
Please, review it. It's only preliminary, I think we can simplify a bit in a couple of places, but first I'd like to see if this, indeed, fixes the issue at hand.

daN4cat commented on application/controllers/Items.php:
```diff
@@ -64,14 +64,23 @@ public function search()
 	
 	public function pic_thumb($pic_filename)
 	{
+	    if($pic_filename=='')
```
can you please fix this and use tabs

daN4cat commented on application/controllers/Items.php:
```diff
@@ -216,9 +225,9 @@ public function view($item_id = -1)
 			// redundant.
 			if(sizeof($images) > 0)
```
this doesn't seem required anymore, please remove the whole statement

daN4cat commented on application/controllers/Items.php:
```diff
@@ -809,6 +818,8 @@ public function do_excel_import()
 	 */
 	private function _update_pic_filename($item)
 	{
+        $filename = pathinfo($item->pic_filename, PATHINFO_FILENAME);
```
tabs

i92guboj commented:
That second commit should fix spacing (I corrected a few more spacing issues while I was at it). And removed some code. A third commit will remove one of the checks, that, at this stage, I think it's unnecessary.

I'll triple test everything, please, do your checks as well before merging.
